src/_gettsim/transfers/wohngeld.py

```diff
@@ -13,37 +21,55 @@
 }
 
 
-def wohngeld_m_hh(
-    wohngeld_nach_vermög_check_m_hh: float,
-    wohngeld_vorrang_hh: bool,
-    wohngeld_kinderzuschl_vorrang_hh: bool,
+def wohngeld_m_wthh(
+    _wohngeld_nach_vermög_check_m_wthh: float,
     erwachsene_alle_rentner_hh: bool,
+    wohngeld_kinderzuschl_vorrang_wthh: bool,
+    wohngeld_vorrang_wthh: bool,
 ) -> float:
-    """Calculate final housing benefit on household level.
+    """Calculate housing benefit.
+
+    Wohngeld has priority over ALG2 if the recipients can cover their needs according to
```

Comment:
This documentation belongs to some meta-document, has nothing to do with this particular function (other than this being step 3, right?)

Reply:
Yes I added it here because this functions yields the "main" target of the module and that way someone who uses the Wohngeld module extensively is likely to stumble over it. But we could also do this in the module docstring if you think that's a better fit?

Reply:
Yeah, sounds like a good place to put it.